sysbuild/Kconfig.mcuboot

```diff
@@ -143,6 +143,13 @@ config MCUBOOT_UPDATEABLE_IMAGES
 	help
 	  The number of images that MCUboot will be built with.
 
+config MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES
+	bool "Sync updateable image value to main application"
+	default y
+	help
+	  If enabled then will synchronise the value of ``MCUBOOT_UPDATEABLE_IMAGES`` to MCUboot
```

Comment:
You mean the `UPDATEABLE_IMAGE_NUMBER` or the MCUmgr Kconfig `MCUMGR_GRP_IMG_UPDATABLE_IMAGE_NUMBER`?
(The `MCUBOOT_UPDATEABLE_IMAGES` is something new? I can not find it...)

Reply:
`MCUBOOT_UPDATEABLE_IMAGES` is the sysbuild setting, if this is set then it will set it to both mcuboot and application, if not then it will only set it to mcuboot... though not sure why the help text mentions mcuboot

Reply:
Have re-read the text and main application is on the line after, so is correct